mmv1/products/privateca/api.yaml

```diff
@@ -773,10 +1063,10 @@ objects:
               name: 'postalCode'
               description: The postal code of the subject.
               input: true
-          - !ruby/object:Api::Type::String
-            name: 'commonName'
-            description: The common name of the distinguished name.
-            required: true
+            - !ruby/object:Api::Type::String
+              name: 'commonName'
```

Comment:
This was dedented one too much before, right?

Reply:
It was moved into the sub-object when the resource went to GA.

The indenting in this file is a little unusual, one indent was used in some places and apparently YAML is fine with that. I tried to fix it where I found it